projects/ngx-datatable/src/lib/components/body/body-row.component.ts

```diff
@@ -61,6 +61,9 @@ import { ColumnGroupWidth, PinnedColumns } from '../../types/internal.types';
   `
 })
 export class DataTableBodyRowComponent<TRow = any> implements DoCheck, OnChanges {
+  private scrollbarHelper = inject(ScrollbarHelper, { skipSelf: true });
```

Comment:
```suggestion
  private scrollbarHelper = inject(ScrollbarHelper);
```

`ScrollbarHelper` is provided in the module. So for me it looks like skipSelf is pointless and we can just remove it